internal/identity/oidc/auth0/auth0.go

```diff
@@ -0,0 +1,49 @@
+// Package auth0 implements OpenID Connect for auth0
+//
+// https://www.pomerium.io/docs/identity-providers/auth0.html
```

Comment:
👀

Reply:
yeah definitely. We can do that in a subsequent PR once we know what will be needed for the service account.

Reply:
@desimone I added a basic markdown guide which we can add to.